### PR TITLE
feat: add variable to override the hostname

### PR DESCRIPTION
**Pull request by LunarWatcher**
TL;DR: Added `g:aw_hostname` to override the hostname. This does not affect the default setting; if `aw_hostname` doesn't exist, it falls back to `hostname()` as usual. Changing the hostname is fully manual, so no instant data breakage (if that's even possible; I don't know) just by updating.

---

I'm stuck on Windows at times, where `hostname()` returns a fully upper-case version of the hostname (on this particular computer, `ARI`). Meanwhile, the other plugins detect the hostname as what I actually inserted when setting it, and one of the two forms I'm fine with; `Ari`. This is Windows/Vim weirdness that the plugin can't really be expected to handle. 

For me, seeing `ARI` and  `Ari` in my dashboard annoys me more than it should, though I'm sure there's people who have other reasons to forcibly override it to fit other conventions, or a different name altogether. Adding a variable lets me override it so I don't want to scream whenever I look at my dashboard and click into anywhere the hostnames are listed.

Additionally, any form of forced convention override in the plugin would have consequences, which a manual variable-based override mitigates by allowing caution prior to changing it. The answer to this question does change drastically if the other plugins that use `Ari` override the case, but I've been using ActivityWatch for about an hour; needless to say I have no clue what conventions are used in other components, nor what the consequences of forcing a hostname change on existing data is.

### CODE REVIEW

**Comment by ErikBjare** [comment]
This is a very niche config option, for a very niche issue, but it's basically 0 code, so nice that it does the job for you :)

Thanks for the contribution! 

Merging.
